blog.html

```diff
@@ -0,0 +1,182 @@
+    <link rel="mask-icon" href="./assets/images/safari-pinned-tab.svg" color="#f18719" />
+    <link rel="shortcut icon" type="image/x-icon" href="./assets/images/favicon.ico" />
+    <meta name="msapplication-TileColor" content="#da532c" />
+    <meta name="theme-color" content="#ffffff" />
+    <meta property="og:image" content="./assets/images/apple-touch-icon.png" />
+    <script src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
+    <script src="./assets/js/siteload.js"></script>
+    <meta name="viewport" content="width = device-width" />
+    <link rel="stylesheet" href="./assets/css/percircle.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+        integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+</head>
+
+<body>
+    <header id="header"></header>
+    <noscript>Please Enable JavaScript to enjoy this website in all its beauty :)</noscript>
+    <div style="width: 100%;">
+        <h1>My Blog:</h1>
+        <div id='toc'>
+            <h2>Table Of Contents</h2>
+            <ul>
+                <li><a href="#PPL">PPL(A)</a></li>
+                <li><a href="#WWDC20">WWDC20</a></li>
+                <li><a href="#WWDC19">WWDC19</a></li>
+                <li><a href="#WWDC18">WWDC18</a></li>
+                <li><a href="#HUGOBOSS">HUGO BOSS AG</a></li>
+                <li><a href="#WWDC17">WWDC17</a></li>
+                <li><a href="#TG">High School</a></li>
+                <li><a href="#KomaNet">KomaNet GmbH</a></li>
+            </ul>
+        </div>
+        <div class="text">
+            <h3 id="PPL">FCL PPL(A)</h3>
+            <a href="https://instagram.com/thisIsTheFoxe">Some highlights on my Instagram</a>
+            <p>
+                Ever since I first visited the West Coast of America, I had the dream of leaving Germany. I don't know
+                why, but it was very hard for me to imagine a future where I would live in the same country for the rest
+                of my life. But after graduating from high school, I was accepted for a cooperative study program.
+                During this time (especially after spending some time in Manchester), my focus shifted from the US to
+                New Zealand. I applied for a working holiday visa and was accepted. Unfortunately, due to COVID-19, I
+                was not able to enter the country as planned after graduating. So I had to spend my time and money doing
+                something else.<br />
+                I always had the dream of flying. But because of my eyesight, I also knew that I would not be able to
+                pursue a career in aviation unless I had eye surgery. However, many of my friends
+                (including <a target="_blank" href="https://www.mbobpro.com/blog/3.html">Marko</a>) inspired me to pursue
```

Comment:
Yo @chih98, you ok with me linking to your site here? ^^

Reply:
yup - thanks for checking with me! 🙂